docker-compose.yml

```diff
@@ -154,7 +157,45 @@ services:
       - 24224:24224
       - 24224:24224/udp
     depends_on:
-      - elasticsearch
+      - kibana
```

Comment:
Elasticsearch should not depend on Kibana.

Reply:
I did this intentionally so that I should not call elastic, kibana and fluentbit seperately.

Reply:
Please remove it and add an additional call for Kibana.